lib/Runtime/Base/ThreadContext.h

```diff
@@ -753,7 +753,7 @@ class ThreadContext sealed :
     // Number of script context attached with probe manager.
     // This counter will be used as addref when the script context is created, this way we maintain the life of diagnostic object.
     // Once no script context available , diagnostic will go away.
-    long crefSContextForDiag;
+    LONG crefSContextForDiag;
```

Comment:
I see, then maybe `ThreadContext.cpp` should have 'LONG lref = InterlockedDecrement(...)`?